packages/frontend/src/components/Layout.js

```diff
@@ -1,10 +1,12 @@
 import React from "react";
 import styles from "./LayoutStyle.module.css";
-import HardcodedTable from "./Table.js";
+import UsersTable from "./Table.js";
 import HardcodedTodoList from "./ToDo";
 import Profile from "./Profile";
 
 const Layout = () => {
+  const token = localStorage.getItem("token");
+  const householdId = "65e93b29d9f311f834219bed"
```

Comment:
Ideally we find a way to get the householdId dynamically, but you explained the situation - my other comment down below could be worth looking at?